app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/landing_page.scala.html

```diff
@@ -123,8 +124,10 @@ <h2 class="heading-medium" id="beforeYouStart">@messages("landingPage.beforeYouS
         <li>@messages("landingPage.beforeYouStart.list1.bullet7")</li>
         <li>@messages("landingPage.beforeYouStart.list1.bullet8")</li>
         <li>@messages("landingPage.beforeYouStart.list1.bullet9")</li>
-    </ul>     
-    
+    </ul>
+    <p>
+        @warning(messages("warning.incorrectcalculation"))
+    </p>
```

Comment:
I think, this <p></p> tag is not required. Please check existing examples. Probably please double check in UI

Reply:
Removed the <p> tag. I wanted to be inline with the existing tags.